src/pages/NotFound/index.tsx

```diff
@@ -1,11 +1,11 @@
 import MainLayout from '../../layouts/MainLayout';
-import utils from '../../styles/modules/utils.module.css';
+import stylesUtils from '../../styles/modules/stylesUtils.module.css';
```

Comment:
Keep `stylesUtils.module.css`. I mean only in naming of import.

If need same naming with file, use `as`. 

```
import utils as styleUtils from '../../styles/modules/utils.module.css';
```

Reply:
Fixed